Monika After Story/game/script-topics.rpy

```diff
@@ -12126,6 +12126,63 @@ label monika_trick_monika:
     m 5eubfb "And I only have eyes for you too!"
     return "derandom|love"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="player_red_poems",
+            category=['literature'],
+            prompt="Red poetry",
+            random=True
+        )
+    )
+
+default persistent._mas_red_poetry = None
+#True if player likes poems, false if not
+label player_red_poems:
+    m 2eub "Say [player]..."
+    m 2eua "You know that I like poetry, but I was wondering something."
+    menu:
+        m 2eua "Did you read poems often before meeting me?"
```

Comment:
```suggestion
        m 2eua "Did you read poems often before you joined the Literature Club?"
```
Makes more sense I think.

Reply:
Yeah, sounds better. I also change the place of the "often".